python/tank/bootstrap/bundle_downloader.py

```diff
@@ -9,8 +9,8 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import os
-from sgtk import hook
-from sgtk import LogManager
+from tank import hook
```

Comment:
Bad!
If the first Toolkit import in a script is import sgtk (like in one of my integration test script), then this import fails because the sgtk module needs the tank module to be fully initialized before being able to assign all the tank attributes to sgtk.